converters/fromZigbee.js

```diff
@@ -4002,7 +4002,7 @@ const converters = {
             });
             dataStr += '\n';
             const fs = require('fs');
-            fs.appendFile('data/tuya.dump.txt', dataStr, (err) => {
+            fs.appendFile('/share/zigbee2mqtt/tuya.dump.txt', dataStr, (err) => {
```

Comment:
This may work for hassio, but will break other setups which don't have `/share`. With what exception does saving to `data/` fail?

Reply:
Zigbee2MQTT:debug 2021-11-10 11:18:29: Received Zigbee message from 'Büro', type 'commandGetData', cluster 'manuSpecificTuya', data '{"data":{"data":[0],"type":"Buffer"},"datatype":1,"dp":1,"fn":0,"status":0,"transid":116}' from endpoint 1 with groupID 0
Error: ENOENT: no such file or directory, open 'data/tuya.dump.txt'

As pointed out, at least adding a '/' in front of 'data/tuya.dump.txt' would fix this for Hassio.

Reply:
What is logged when you add `console.log(__dirname);` just above `fs.appendFile`?

Reply:
nothing is logged; interestingly the error is also not logged in the log files, but shows only  when directly viewing the log from the addon

Reply:
What if use ./data/tuya.dump.txt?

Reply:
Wouldn't it be possible to catch the error and try the next possible location? Maybe it is also nicer to console.log the error, because the current method crashes the complete container. I tried implementing it but my JS skills are not sufficient.